.github/workflows/tenzir.yaml

```diff
@@ -846,6 +852,8 @@ jobs:
           - {setup: {os: macos-latest}, plugin: {name: Netflow}}
           - {setup: {os: macos-latest}, plugin: {name: Pipeline Manager}}
           - {setup: {os: macos-latest}, plugin: {name: Sigma}}
+          - {setup: {os: macos-latest}, plugin: {name: Kafka}}
+          - {setup: {os: macos-latest}, plugin: {name: ZeroMQ}}
```

Comment:
Nit: both of the lists here used to be alphabetically sorted.

Reply:
34a3e812af